helix-core/src/main/java/org/apache/helix/util/InstanceValidationUtil.java

```diff
@@ -138,12 +149,10 @@ public static boolean isAlive(HelixDataAccessor dataAccessor, String instanceNam
    * Method to check if the instance is assigned at least 1 resource, not in a idle state;
    * Independent of the instance alive/enabled status
    * @param dataAccessor
-   * @param clusterId
    * @param instanceName
    * @return
    */
-  public static boolean hasResourceAssigned(HelixDataAccessor dataAccessor, String clusterId,
-      String instanceName) {
+  public static boolean hasResourceAssigned(HelixDataAccessor dataAccessor, String instanceName) {
```

Comment:
Since this is a public method, can we consider deprecating it and re-creating the version without `clusterId` as an argument?

Reply:
Good point, updated. Thanks.